Tools/build/deepfreeze.py

```diff
@@ -270,6 +272,8 @@ def generate_code(self, name: str, code: types.CodeType) -> str:
             self.write(f".co_nplaincellvars = {nplaincellvars},")
             self.write(f".co_ncellvars = {ncellvars},")
             self.write(f".co_nfreevars = {nfreevars},")
+            self.write(f".co_version = {next_code_version},")
+            next_code_version += 1
```

Comment:
assert that it's less than 10000, or is this crazy?

Reply:
The 10000 is just a hack. I'll fix it properly.